### PR TITLE
Migrate Openshift Serverless Logic to Kogito SW 1.30.x

### PR DESCRIPTION
**Pull request by akumar074**
[1.30.x] Update dev ui start workflow page

[1.30.x] Update swf diagram docs

Updated Kogito_sw_examples_urls

KOGITO-7975 Removed workflowdata property from requests

Docs Patch PR for KOGITO-8078: Document adding custom type through addon

[KOGITO-7951] Adding custom types through addons

KOGITO-8145 Fix broken links in SWF Guide - section Security

Orchestrating third party services with oauth2 updates

Updating icon for timeout support in CNCF section

Docs Patch PR for KOGITO-7967: Improve clarity of procedure to start gRPC application in the guide

KOGITO-7967 Improve clarity of procedure to start gRPC application in the guide

KOGITO-8039 Changed link to example and added instruction about REST

[KOGITO-7764] Timeout support

KOGITO-7756 Unified groupIds that are used when creating the project

KOGITO-7053: SWF Viewer - Community Documentation

### CODE REVIEW

File: antora.yml [hunk 1]
```diff
@@ -22,10 +22,13 @@ asciidoc:
     kie_tools_node_min_version: 16.13.2
```
Comment (ricardozanini) [inline antora.yml:22]: We must hold this PR until we have the final version for `kogito_version_redhat`, `quarkus_version`, `vscode_version`, and `kn_cli_version`. @MarianMacik @akumar074

Comment (ricardozanini) [reply antora.yml:22]: cc @ederign

**Review by radtriste — APPROVED** [review]

File: antora.yml (hunk 1 above)
Comment (ricardozanini) [reply antora.yml:22]: @akumar074 can you please update this PR with the following versions?

`kogito_version_redhat: 1.30.0.Final-redhat-00001`
`kn_cli_version : 0.25.0`

Comment (akumar074) [reply antora.yml:22]: @ricardozanini Yes, sure.
and will `quarkus_version` and `vscode_version` remain same?

Comment (ricardozanini) [reply antora.yml:22]: Yes! @paulovmr is there any updates in the VSCode Version?

Comment (paulovmr) [reply antora.yml:22]: @ricardozanini currently the supported VSCode version for the extensions is 1.66.0 or newer.

File: antora.yml [hunk 2]
```diff
@@ -18,14 +18,17 @@ asciidoc:
     graalvm_min_version: 21.3.0
     spec_version: 0.8
     vscode_version: 1.46.0
```
Comment (ricardozanini) [inline antora.yml:20]:
```suggestion
    vscode_version: 1.66.0
```

**Review by paulovmr — APPROVED** [review]

**Review by MarianMacik — APPROVED** [review]

**Pull request merged**